README.md

```diff
@@ -1,12 +1,15 @@
 # fmralign
 
+![build](https://img.shields.io/github/actions/workflow/status/parietal-inria/fmralign/testing.yml?event=push&style=for-the-badge)
+![python version](https://img.shields.io/badge/python-3.9_|_3.10_|_3.11|_3.12|_3.13-blue?style=for-the-badge)
```

Comment:
Not supporting 3.13 is fine for me (nilearn doesn't officially support it either). We can remove the badge for now.
```suggestion
![python version](https://img.shields.io/badge/python-3.9_|_3.10_|_3.11|_3.12-blue?style=for-the-badge)
```